run-tests.js

```diff
@@ -12,12 +12,12 @@ import ansi from 'ansicolor'
 import log from 'ololog'
 import ps from 'child_process'
 ansi.nice
-/*  --------------------------------------------------------------------------- */
+//  --------------------------------------------------------------------------- //
```

Comment:
what;s the problem with those comments?

Reply:
typically, I don't think it's a good practice to use multiline comment for single-line.  second, in real-world cases, when you need to comment out some part of file with multi-line comments, such inline-multilines make conflicts.
but in this case, if you think them to stay, i'll leave them as is.